trollflow2/launcher.py

```diff
@@ -112,7 +113,8 @@ def check_results(produced_files, start_time, exitcode):
             LOG.critical('Process crashed with exit code %d', exitcode)
     if not error_detected:
         elapsed = end_time - start_time
-        LOG.info('All files produced nominally in %s.', str(elapsed), extra={'time': elapsed})
+        LOG.info(f'All {qsize:d} files produced nominally in '
+                 f"{elapsed!s}")
```

Comment:
Could you put `extra` back in? we put it there for monitoring purposes

Reply:
Sure.  I couldn't reproduce what it was doing, though; there was one `%s` in the previous log message but two additional arguments, so it appeared it was not doing anything?

Reply:
`extra` is just passed to the logrecord as is, to be used later by formatters or handlers. A custom handler would be able to use that info for eg monitoring purposes. See the paragraph on `extra` here: https:/https://docs.python.org/3/library/logging.html#logging.Logger.debug

Reply:
I see.  I thought it was missing a corresponding format key in the string.  I've put it back now.